public/pages/MonitorDetails/containers/MonitorDetails.js

```diff
@@ -299,7 +476,28 @@ export default class MonitorDetails extends Component {
               </EuiFlexItem>
             ) : null}
           </EuiFlexItem>
-
+          <EuiFlexItem grow={false}>
+            <EuiToolTip
+              position="top"
+              content={
+                monitor.ui_metadata.search.searchType === 'graph'
+                  ? ''
+                  : 'Anomaly detector can only be automatically created from monitor definition type "visual editor"'
+              }
+            >
+              <EuiButton
+                isLoading={updating}
+                onClick={() => this.convertToADConfigs(this.state.monitor)}
+                disabled={
+                  monitor.ui_metadata.search.searchType !== 'graph' ||
+                  monitor.ui_metadata.search.fieldName === '' ||
+                  monitor.ui_metadata.schedule.timezone
+                }
+              >
+                Create Detector
```

Comment:
Change to "Create detector" to keep consistent with "Create monitor". Same for other places.

Reply:
fixed